frontend/src/routes/Video.tsx

```diff
@@ -354,17 +354,12 @@ const EmbedCode: React.FC<EmbedCodeProps> = ({ event: { opencastId: id } }) => {
 
     const embedCode = `<iframe ${[
         `src="${CONFIG.opencast.presentationNode}/play/${id}"`,
-        "allowfullscreen",
         `style="${[
             "border: none;",
             "width: 100%;",
             "aspect-ratio: 16/9;",
         ].join(" ")}"`,
         'name="Player"',
-        'scrolling="no"',
-        'frameborder="0"',
-        'marginheight="0px"',
-        'marginwidth="0px"',
```

Comment:
Ok but... are all these things the default in new browsers? If not, shouldn't they be replaced with anything? Like `allow="fullscreen"` or sth?

Reply:
* I added `allow="fullscreen"`
* `frameborder` has its alternative in the CSS (`border: none;`)
* `scrolling` is `auto` by default, so as long as we make sure the content fits in the `iframe` it shouldn't be necessary. And in fact I think it's better to allow scrolling in case we fail to make sure, but it's not a hard preference. What do you think?
* `margin{width,height}` are actually discouraged because they modify the embedded page. (The CSS equivalent would be to set the `margin` of the embedded page's `body`.)